app/controllers/state_file/questions/nj_household_rent_own_controller.rb

```diff
@@ -2,10 +2,8 @@ module StateFile
   module Questions
     class NjHouseholdRentOwnController < QuestionsController
       include ReturnToReviewConcern
-      def edit
-        @filing_year = Rails.configuration.statefile_current_tax_year
-        super
-      end
+
+      before_action -> { @filing_year = Rails.configuration.statefile_current_tax_year }
```

Comment:
[DUST] Flagging that we have an open discussion on how to source `@filing_year` in this PR: https://github.com/codeforamerica/vita-min/pull/4800#pullrequestreview-2332008557

Reply:
@jachan it seems like that's talking about a separate value? This value is coming from the Rails config. Is there something I'm missing?

Reply:
I guess it's `return_year` vs `filing_year` which are separate, got the two mixed up! We can disregard.